shared/wallets/search/index.js

```diff
@@ -1,50 +0,0 @@
-// @flow
```

Comment:
The search story got merged into the `SendForm/participants` story if you're looking for it.